examples/async/babel.config.js

```diff
@@ -14,9 +13,7 @@ module.exports = {
       },
     ],
     '@babel/react',
-    '@babel/stage-2',
+    ['@babel/stage-2', { decoratorsLegacy: true }],
```

Comment:
do we even use decorators anywhere?

Reply:
We should remove `stage-2` preset anyway and use specific transforms if needed. I suspect the only thing we really use is `@babel/proposal-object-rest-spread`